src/utils/performance_optimizer.py

```
@@ -9,203 +1,65 @@
-    async def track(self, metric: str, value: float) -> None:
-        """Record system metrics"""
-        self.metrics[metric].append((time.monotonic(), value))
-
-    async def get_realtime_stats(self) -> Dict[str, float]:
-        """Generate performance statistics"""
         return {
-            'throughput_1m': self._calculate_throughput(60),
-            'latency_p95': self._calculate_percentile('latency', 95),
-            'error_rate_1m': self._calculate_error_rate(60)
+            'cpu': cpu_percent,
+            'memory': memory_percent
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)